lib/factory_bot/strategy/stub.rb

```diff
@@ -76,6 +76,10 @@ def destroyed?
             def created_at
               @created_at ||= Time.now.in_time_zone
             end
+
+            def created_at=(date)
```

Comment:
Is there a way we could deal with arguments of the wrong type, like `stub.created_at = 'nonsense'`? The original `created_at` would type cast that string to `nil`, but this method will preserve whatever we pass in.

Reply:
Or I wonder if we could get rid of the custom setter and just have created at return `super` if it is present?